osm_fieldwork/CSVDump.py

```diff
@@ -326,15 +322,21 @@ def main():
 
     # if verbose, dump to the terminal.
     if args.verbose is not None:
```

Comment:
Hey @spwoodcock , I have made the changes now.

The log_level and log_stream variables are now replaced with hardcoded basicConfig versions.

Because the urllib configuration depends on log_level, I moved it to the args.verbose block. 

I left the ODKCentral.py code because it doesnt have a verbose argument and instead depends on an environment variable.